app/src/DesignSystem.re

```diff
@@ -11,3 +15,35 @@ let margin = (top, right, bottom, left) =>
     marginBottom(px(bottom)),
     marginLeft(px(left)),
   ]);
+
+module Flex = {
```

Comment:
Is it significant that this module is distinct from `Layout`, or can we treat everything in here as layout primitives as well?

Reply:
No, you're right, actually everything falls into place if I move it into layout.